lib/ZwaveClient.js

```diff
@@ -1899,12 +1904,33 @@ ZwaveClient.prototype.writeValue = async function (valueId, value) {
   if (this.driverReady) {
     let result = false
 
-    if (valueId.type === 'number' && typeof value === 'string') {
+    if (
+      valueId.type === 'number' &&
+      typeof value === 'string' &&
+      !isNaN(value)
+    ) {
       value = Number(value)
     }
 
     try {
-      result = await this.getNode(valueId.nodeId).setValue(valueId, value)
+      const zwaveNode = await this.getNode(valueId.nodeId)
+
+      // handle multilevel switch 'start' and 'stop' commands
+      if (
+        valueId.commandClass === CommandClasses['Multilevel Switch'] &&
+        isNaN(value)
+      ) {
+        if (/stop/i.test(value)) {
+          await zwaveNode.commandClasses['Multilevel Switch'].stopLevelChange()
```

Comment:
@AlCalzone here is where I call it.

Reply:
Do you get any specific error? Lots of things can go wrong.

Reply:
`stopLevelChange failed` that's the error :man_shrugging:

Reply:
Ive attached logging to the issue: https://github.com/zwave-js/zwavejs2mqtt/issues/30#issuecomment-736558562

```
14:31:50.571 DRIVER » [Node 004] [REQ] [SendData]
                      │ transmit options: 0x25
                      │ callback id:      73
                      └─[SupervisionCCGet]
                        │ session id:      10
                        │ request updates: false
                        └─[MultilevelSwitchCCStopLevelChange]
```

Reply:
I don't think I have made something wrong becasue from the logs seems the call is done correctly but something else has failed

Reply:
```
14:31:50.571 DRIVER » [Node 004] [REQ] [SendData]
                      │ transmit options: 0x25
                      │ callback id:      73
                      └─[SupervisionCCGet]
                        │ session id:      10
                        │ request updates: false
                        └─[MultilevelSwitchCCStopLevelChange]
14:31:50.571 DRIVER   send thread state: sending.execute
14:31:50.574 SERIAL « [ACK]                                                                   (0x06)
14:31:50.575 DRIVER   send thread state: sending.execute
14:31:50.578 SERIAL « 0x0104011301e8                                                       (6 bytes)
14:31:50.578 SERIAL » [ACK]                                                                   (0x06)
14:31:50.579 DRIVER « [RES] [SendData]
                        was sent: true
14:31:50.581 DRIVER   send thread state: sending.execute
14:31:50.593 SERIAL « 0x0107001349000002a0                                                 (9 bytes)
14:31:50.594 SERIAL » [ACK]                                                                   (0x06)
14:31:50.595 DRIVER « [REQ] [SendData]
                        callback id:     73
                        transmit status: OK
14:31:50.598 DRIVER   send thread state: sending.execute
14:31:50.598 DRIVER   send thread state: sending.waitForUpdate
14:31:50.608 SERIAL « 0x010b00040004056c020a000095                                        (13 bytes)
14:31:50.609 SERIAL » [ACK]                                                                   (0x06)
14:31:50.611 DRIVER « [Node 004] [REQ] [ApplicationCommand]
                      └─[SupervisionCCReport]
                          session id:          10
                          more updates follow: false
                          status:              NoSupport
                          duration:            [Duration: 0seconds]
```

>  status:              NoSupport

Not supported from the device?

Reply:
I found it:
```
14:31:50.611 DRIVER « [Node 004] [REQ] [ApplicationCommand]
                      └─[SupervisionCCReport]
                          session id:          10
                          more updates follow: false
                          status:              NoSupport
                          duration:            [Duration: 0seconds]
```
where `NoSupport` means "The command is not supported by the receiver."

Reply:
Which is weird because `MultilevelSwitchCCStopLevelChange` is a V1 command.

Reply:
We could try opting out of using supervision, but I'd like to test if this is the problem first.

@robertsLando Can you try to send the command without going through the API:
```ts
		const cc = new MultilevelSwitchCCStopLevelChange(this.driver, {
			nodeId: /* The node's ID */,
			endpoint: /* The endpoint (or 0) */,
		});

		await this.driver.sendCommand(cc);
```

Reply:
@AlCalzone Where do I found that `MultilevelSwitchCCStopLevelChange`?

Reply:
General thought: from what I have undestanded most times do 'support checkes' seems isn't working most times even if it seems correct from a logical point of view. What about skip this kind of check and simply do things and return an error if they fails?

Reply:
Except the support check happens on the device in this case. It supports Supervision and responds to our request, telling us that `MultilevelSwitchCCStopLevelChange` is not supported (which I don't believe).

> What about skip this kind of checka and simply do things and return an error if they fails?

No. Using an unsupported command means we have to wait seconds to be sure that there is no response. If https://github.com/zwave-js/node-zwave-js/issues/997 happens, using an unsupported command means we'll wait 10 seconds when doing that.

Reply:
> Where do I found that MultilevelSwitchCCStopLevelChange?

Doesn't VSCode offer you an auto-import? Should be exported from `zwave-js/CommandClass`

Reply:
> Doesn't VSCode offer you an auto-import? Should be exported from zwave-js/CommandClass

Nope, cannot find it, are you sure it's `zwave-js/CommandClass`?

Reply:
> Doesn't VSCode offer you an auto-import? Should be exported from zwave-js/CommandClass

Nope, dunno why :man_shrugging: Anyway I have done now

Reply:
@ODINServ COuld you try now?

Reply:
Got a different error:
```
  z2m:Mqtt Message received on zwavejs2mqtt/maarten_right/switch_multilevel/endpoint_1/targetValue/set +7s
  z2m:Zwave Error while writing stop on 4-38-1-targetValue: MultilevelSwitchCCStopLevelChange is not a constructor +4s
  z2m:Zwave Unable to write stop on 4-38-1-targetValue +0ms
```

Reply:
:exploding_head:

Reply:
@AlCalzone ?

Reply:
Ah shit, it is not exported from there. Try that path `zwave-js/build/lib/commandclass/MultilevelSwitchCC`.

Reply:
Fixed

Reply:
@ODINServ Could you try now

Reply:
It works!! :smiley: 

Your are my heroes! :muscle: 
I found in the mean time more users that want this, so your there heroes as well.
(https://community.home-assistant.io/t/fibaro-roller-shutter-3-fgr-223-cannot-get-it-to-work-properly/97577/115 )

:beers: :clinking_glasses: 

If you are interested in the logs: https://github.com/zwave-js/zwavejs2mqtt/issues/30#issuecomment-736645024

Reply:
@AlCalzone So based on this I think you will have to remove the check and just send the command? Can I revert to the previous version and use start/stopLevelChange?

Reply:
Ok, I'll need to figure out why I used Supervision for those commands in the first place.

Reply:
@ODINServ start spam hass forums with zwavejs so :smile:

Reply:
Hmm, fuck past me.
I changed this [here](https://github.com/zwave-js/node-zwave-js/pull/648/files) but I no longer know why...